src/components/AgentDashboard(Rent)/ChangeStatus.js

```diff
@@ -324,16 +358,16 @@ function ChangeStatus() {
           >
             <p className="text-[1.2rem] font-bold">Close Listing</p>
             <div className="flex justify-center  items-center flex-col w-[75%] py-[1rem] gap-y-[1.5rem]">
-            {propertyDetails.status === 'verified' && (  <CommonTopButton
+            {(propertyDetails.status === 'Verified' && isShared) && (  <CommonTopButton
                 bgColor={
-                  closeListingReason === "Rented on B8R" ? "#52796F" : "#D2D7D6"
+                  closeListingReason === "Rented of B8R" ? "#52796F" : "#D2D7D6"
                 }
                 borderColor="#DAF0EE"
                 color={
-                  closeListingReason === "Rented on B8R" ? "#FFFFFF" : "#77A8A4"
+                  closeListingReason === "Rented of B8R" ? "#FFFFFF" : "#77A8A4"
```

Comment:
these should be "Rented on B8R" - @PrakharB8R, any reason you are replacing it? was this change approved?